packages/shortest/src/core/runner/index.ts

```diff
@@ -61,29 +60,6 @@ export class TestRunner {
     this.browserManager = new BrowserManager(this.config);
   }
 
-  private async findTestFiles(pattern?: string): Promise<string[]> {
-    this.log.trace("Finding test files", { pattern });
-    const testPattern = pattern || this.config.testPattern || "**/*.test.ts";
```

Comment:
Config should be the source of truth of all default values, in conjuction with Zod validation.